Makefile

```diff
@@ -18,7 +18,6 @@ cov cover coverage: pre-commit
 
 mototest:
 	python -Wd -X tracemalloc=5 -X faulthandler -m pytest -vv -m "not localonly" -n auto --cov-report term --cov-report html --cov-report xml --cov=aiobotocore --cov=tests --log-cli-level=DEBUG $(FLAGS) aiobotocore tests
-	@echo "open file://`pwd`/htmlcov/index.html"
```

Comment:
Is this cosmetic, i.e. reduces clutter in standard output? Ideally, it would be in a separate PR ;-)

Reply:
it's a security issue, we should not be calling "open" on a url during the build.  It's so minor not sure it warrants another PR